helmchart/sdp/templates/02.4-deployment-dashboard.yaml

```diff
@@ -0,0 +1,72 @@
+{{- if .Values.dashboard.enabled -}}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "sdp.fullname" . }}-dashboard
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "sdp.labelsWithSuffix" (list . "-dashboard") | nindent 4 }}
+  {{- if .Values.dashboard.deployment.annotations }}
+  annotations:
+    {{- tpl (toYaml .Values.deployment.annotations) . | nindent 4 }}
+  {{- end }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "sdp.selectorLabelsWithSuffix" (list . "-dashboard") | nindent 6 }}
+
+  {{- if .Values.dashboard.deployment.strategy }}
+  strategy:
+    {{- toYaml .Values.dashboard.deployment.strategy | nindent 4 }}
+  {{- end }}
+
+  template:
+    metadata:
+      {{- if .Values.dashboard.deployment.podAnnotations }}
+      annotations:
+        {{- tpl (toYaml .Values.dashboard.deployment.podAnnotations) . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "sdp.selectorLabelsWithSuffix" (list . "-dashboard") | nindent 8 }}
+    spec:
+      {{- if .Values.serviceAccount.name }}
+      serviceAccountName: {{ tpl .Values.serviceAccount.name $ }}
+      {{- end }}
+      securityContext:
+        {{- toYaml .Values.dashboard.deployment.podSecurityContext | nindent 8 }}
+
+      containers:
+      # ============================= Stellar Disbursement Platform: =============================
+        - name: {{ .Chart.Name }}-dashboard
+          securityContext:
+            {{- toYaml .Values.dashboard.deployment.securityContext | nindent 12 }}
+          image: "{{ .Values.dashboard.image.fullName }}"
+          imagePullPolicy: {{ .Values.dashboard.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: {{ include "dashboard.port" . }}
```

Comment:
does this contain a default value. if dashboard.port is not defined it will cause an error

Reply:
Yes, the default value is `80`:
https://github.com/stellar/stellar-disbursement-platform-backend/blob/338c3c7c79a472557b573271c3665950edd05022/helmchart/sdp/templates/_helpers.tpl#L185-L190